eng/dependabot/Packages.props

```diff
@@ -26,7 +26,7 @@
     <PackageReference Update="Microsoft.NET.Test.Sdk" Version="17.4.1" />
     <PackageReference Update="xunit.abstractions" Version="2.0.3" />
     <PackageReference Update="Newtonsoft.Json.Schema" Version="3.0.14" />
-    <PackageReference Update="Verify.XUnit" Version="19.5.0" />
-    <PackageReference Update="Verify.DiffPlex" Version="2.0.1" />
+    <PackageReference Update="Verify.Xunit" Version="19.3.0" />
+    <PackageReference Update="Verify.DiffPlex" Version="1.3.0" />
```

Comment:
Would DiffPlex 2.0.0 be possible?
1.3.0 is not presenting whitespace-only changes in understadable way - so it led to quite a few confusions during CIs (especially when snapshot file was edited in VS which auto-trimmed whitespaces)